UserContext/AnonymousRequestMatcher.php

```diff
@@ -11,27 +11,27 @@
 
 namespace FOS\HttpCacheBundle\UserContext;
 
+use FOS\HttpCache\UserContext\AnonymousRequestMatcher as BaseAnonymousRequestMatcher;
 use Symfony\Component\HttpFoundation\Request;
-use Symfony\Component\HttpFoundation\RequestMatcherInterface;
 
 /**
  * Matches anonymous requests using a list of identification headers.
  */
-class AnonymousRequestMatcher implements RequestMatcherInterface
+class AnonymousRequestMatcher extends BaseAnonymousRequestMatcher
```

Comment:
please add a `@deprecated` annotation saying to use the class from the library instead